Tests/FunctionalTests/Sources/FTRMultiFingerSwipeTest.m

```diff
@@ -0,0 +1,163 @@
+//
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "FTRBaseIntegrationTest.h"
+
+@interface FTRMultiFingerSwipeTest : FTRBaseIntegrationTest
+@end
+
+@implementation FTRMultiFingerSwipeTest
+
+- (CGPoint)referencePoint {
+    return CGPointMake(150, 125);
+}
+    
+- (void)setUp {
+    [super setUp];
+    [self openTestViewNamed:@"Multi finger swipe gestures"];
+}
+    
+- (GREYMultiFingerSwipeAction *)swipeActionForDirection:(GREYDirection)direction usingNumberOfFingers:(NSUInteger)fingerCount {
```

Comment:
Break this method definition please.

- (GREYMultiFingerSwipeAction *)swipeActionForDirection:(GREYDirection)direction 
                                                           usingNumberOfFingers:(NSUInteger)fingerCount